pkg/apis/experimental/v1alpha1/custom_builder_types.go

```diff
@@ -23,7 +23,7 @@ type CustomBuilder struct {
 
 type CustomBuilderSpec struct {
 	Tag   string       `json:"tag"`
-	Stack Stack        `json:"stack"`
+	Stack string       `json:"stack"`
```

Comment:
I wonder if we should rename these fields to `StackRef` and `StoreRef` as they actually refer to kubernetes objects of that type